### ISSUE TITLE
October Release Notes seem to delay or break initialization

### ISSUE THREAD

**Issue by tomwheeler** [gitpod-io/gitpod#14541]
### Bug description

My workspace is created from [this repo](https://github.com/temporalio/edu-101-go-code/) and defines tasks to display a README.md file and open the Simple Browser at startup. This has worked fine for several months, but this evening I found that my workspace now shows the "October GitPod Release 2022" page at startup instead. 

There are two problems here. We're using GitPod to provide an exercise environment for a training course, so the release notes are an unwanted distraction. I thought I had [suppressed pages like that](https://github.com/gitpod-io/gitpod/issues/12183), so I don't think this page should be there in the first place. However, the bigger problem is that the display of this page seems to causing problems with the initialization of the workspace. I originally described this in [a Discord thread](https://discord.com/channels/816244985187008514/1026792978854973460/threads/1039708198564794368), but in summary, the initialization commands running in the terminal do not complete until I close the release notes page, and only at that point is the README.md file finally displayed. The Simple Browser is never displayed at all, although it works fine when I manually open it by clicking the port number in the bottom and selecting Preview. Since this is for an intro-level training course, so the people using it won't know to do that at first.

I never observed this behavior prior to this evening, but since then, I have created a workspace six times in a row and every single one of them exhibits this behavior. I do not see any errors in my GitPod workspace that indicate a problem, so I'm not sure how to solve it. Discord user GHOST said, "I tried opening your repo and I didn't get the release notes. The README opened but the simple browser didn't open until I closed the readme." 

### Steps to reproduce

1. Create a new workspace from [this repo](https://github.com/temporalio/edu-101-go-code/)
2. Observe that the unwanted "October GitPod Release 2022" page is displayed at startup (as depicted below)
3. Observe that the initialization tasks running in the terminal near the lower-right corner appear to be paused
4. Wait for several minutes to see if the problem resolves itself. When you are convinced that it will not, close the release notes
5. Observe that only at this point is the README.md file displayed in the editor
6. Observe that the Simple Browser is never displayed at all

![image](https://user-images.githubusercontent.com/2183904/200717754-d514930b-ded5-4711-9671-b72a696dd263.png)

### Expected behavior

The workspace should look like this on startup, as it did prior to tonight (note: I last created a workspace five days ago and it worked fine then). There are three important things to observe here, as compared to the observed behavior screenshot above:

1. The commands running in the terminal window have completed
2. The README.md file is displayed
3. The Simple Browser window is displayed

![image](https://user-images.githubusercontent.com/2183904/200717701-3c913613-aea6-48f5-8af4-120b0e72beec.png)

### Example repository

https://github.com/temporalio/edu-101-go-code/

**Comment by axonasif** [comment]
Related: https://github.com/gitpod-io/gitpod/issues/12704

**Comment by axonasif** [comment]
`gp open README.md` is probably unable to detect IDE ready state, so it's blocking further execution.

**Comment by jeanp413** [comment]
Can you repro this consistently? I wasn't able to repro this with latest release

**Comment by tomwheeler** [comment]
I could reproduce it 100% of the time when I reported it (I recall trying it at least 7 times over a 24-hour period, using two different browsers, and the workspace failed to properly initialize in all cases). However, I just tried it again and it's working. 

To be clear, the release notes are still being displayed (which is a problem because unexpected windows like that are an unwelcome distraction that confuses the end users), but _at this moment_, the workspace otherwise does seem to initialize properly. I will add that, unlike before, my README file is being shown (and is the active component in the editor area, as the release notes are shown in the tab that does not have focus).
